tests/.wickerconfig.test.json

```diff
@@ -17,6 +17,8 @@
     "timeout": 150,
     "retry_backoff":5,
     "retry_delay_s": 4
-  }
-  
+  },
+  "gcloud": {
+    "dataset_bucket": "fake_bucket"
```

Comment:
@pickles-bread-and-butter we should definitely support reading or writing a path within the bucket (i.e. similar to `s3_datasets_path`, we could have `gs_datasets_path` that looks like `gs://bucket_name/folder_path`.